rest/client.py

```diff
@@ -44,8 +56,21 @@ def order_book(self, currency_pair: str, group: int = 1):
         if group not in [0, 1, 2]:
             raise ValueError('Group parameter should be 0, 1 or 2.')
 
-        rsp = requests.get(url=self.base_endpoint + '/order_book/' + currency_pair, params={'group': group})
-        if rsp.status_code == 404:
-            raise ValueError('Order book for requested currency pair not found.')
+        data = {'group': group}
+        return self._make_request('GET', '/order_book/' + currency_pair, data)
+
+
+    def transactions(self, currency_pair: str, time: str = "hour"):
```

Comment:
This isn't "time" but rather period. And it should be defined as: period: str = 'hour'. time is weird naming for that.

Reply:
I concur, but this was done following Bitstamp api documentation, which uses "time" for this parameter, was done for the sake of consistency with the API.

![image](https://user-images.githubusercontent.com/52085374/119257757-fe578780-bbc6-11eb-9cdd-e583ec35482f.png)

However, I agree time is weird naming and renamed to period.
Please resolve if you see fit.